changes/2821.changed

```diff
@@ -0,0 +1,4 @@
+Adds navigation.footer to mkdocs features to enable navigation in the footer.
+Updates installation documentation into a single path, using tabs for MySQL/PostgeSQL & Ubuntu/Debian/RHEL.
+Removes centos.md and ubuntu.md as these are collapsed into single `install_system.md` file.
+Updates docker-compose standalone to docker compose plugin to assist in development environment.
```

Comment:
Was it intentional to include all of #4287 in this PR? Should we close that PR in favor of this one, or can they be separated for individual review?

Reply:
If I can move to using #4287 instead, I am good. The update that is needed for this is basically on top of #4287 due to the nature of where the requirement to do the install of NodeJS to. I will gladly re-submit a new PR if we are good that 4287 is set for the release.

Reply:
Which is probably better to get that merged in and then at this point have me submit new work on top of that. Since the linting failures are solved there.